### PR TITLE
presets: set archive when modifying cvar, also drop unimplemented MSAA, fix Unvanquished/Unvanquished#2340

### PR DESCRIPTION
**Pull request by illwieckz**
Set archive when modifying cvar

- fix Unvanquished/Unvanquished#2340

Also reset `r_ext_multisample` which isn't implemented yet, since multisampling isn't implemented yet it means the values were never tested and then, if we set or keep a value set to this cvar, people may have the bad surprise of a potential performance hit the day they upgrade the engine to a version that implements the feature.

### CODE REVIEW

**Comment by illwieckz** [comment]
For `r_ext_multisample` we may prefer to entirely remove it from presets and add a code in Dæmon to reset it, code we would remove the day we implement multisampling.

**Comment by slipher** [comment]
LGTM

**Comment by illwieckz** [comment]
See also this PR to reset `r_ext_multisample` on engine side:

- https://github.com/DaemonEngine/Daemon/pull/773

Anyway, doing it on engine side doesn't prevent us to also do it on preset side.

File: presets/graphics/high.cfg [hunk 1]
```diff
@@ -11,7 +11,7 @@ seta r_heatHaze 1
 seta r_bloom 1
 seta r_subdivisions 4
 seta r_imageMaxDimension 2048
-seta r_ext_multisample 8
+reset r_ext_multisample
```
Comment (slipher) [inline presets/graphics/high.cfg:14]: Now you can drop this

Comment (illwieckz) [reply presets/graphics/high.cfg]: I just done it.

**Comment by illwieckz** [comment]
I added a commit to also remove the MSAA option from the UI.

File: ui/options_graphics.rml [hunk 2]
```diff
@@ -76,22 +76,9 @@
...
-						<option value="8">8×</option>
-						<option value="16">16×</option>
-					</select>
-					<h3><translate>Multisampling level (MSAA)</translate></h3>
-					<p><translate>Anti-aliasing smoothes out “jaggy edges“ but is very costly to your frame-rate.</translate></p>
-				</row>
-
 				<row>
 					<input cvar="r_FXAA" type="checkbox" />
 					<h3><translate>Fast approximate anti-aliasing (FXAA)</translate></h3>
```
Comment (necessarily-equal) [inline ui/options_graphics.rml:81]: Is this one implemented?

**Review by necessarily-equal — APPROVED** [review]
One question, LGTM because it improves things

File: ui/options_graphics.rml (hunk 2 above)
Comment (illwieckz) [reply ui/options_graphics.rml:81]: Yes, it doesn't work as expected but it is implemented:

- https://github.com/DaemonEngine/Daemon/issues/533

**Comment by slipher** [comment]
LGTM

**Pull request merged**